src/components/Timeline.js

```diff
@@ -68,7 +68,7 @@ function Timeline() {
     end: Date.now(),
   });
   const { map } = useMapContext();
-  const isGlobalPage = !!router.asPath.match(/\/top/g);
+  const isGlobalPage = !!router.pathname.match(/^(?:\/top|\/)$/g);
```

Comment:
@RubenSmn we need to do some extra work on these cases, the path on other envs will be a bit different with local, the root of next.js will be: /web-map-beta/beta as the base path, os we will fail on your code above on other envs, please check my code here to handle it: https://github.com/Greenstand/treetracker-web-map-client/blob/752f85020c90473c4fd56940edb6f6ef82ca58b6/src/components/App.js#L76-L90

Reply:
Thanks for the extra info!